docs/narr/viewconfig.rst

```diff
@@ -1082,9 +1084,30 @@ For example:
 
 In this case, the ``application/json`` view should always be selected in cases where it is otherwise ambiguous.
 
+.. _default_accept_ordering:
```

Comment:
Above this line, suggest an optional index entry:

```
.. index::
    single: default accept ordering
```

Reply:
I'm not sure whether we need to use a `.. versionchanged:: 1.10` or `.. versionadded:: 1.10` somewhere in these changes, too.